src/components/Message/MessageSimple/MessageContent.tsx

```diff
@@ -174,7 +191,6 @@ const MessageContentWithContext = <
     FileAttachmentGroup = DefaultFileAttachmentGroup,
     Gallery = DefaultGallery,
     Giphy,
-    Message,
```

Comment:
do we want to do a similar thing here for what you did for Attachment so we support it as a prop or from context?

Reply:
send this component has already been rendered by now at the MessageSimple level, we're only passing this through to get it into the props of the MessageText custom component

Reply:
yeah but now it won't work for clients who had been passing in this prop here for changes at the MessageText level, they'll have to use context and it'll be a breaking change for them?

Reply:
well that would have been an issue before anyways cus this was only getting as a prop because it was a WithContext memoized component, so was still coming from context essentially at the parent level

Reply:
but is that only if they used the default instead of customizing like `<MessageSimple MessageContent={() => <CustomMessageContent Message={CustomMessage}/>} />`?